common/djangoapps/util/date_utils.py

```diff
@@ -89,7 +103,7 @@ def from_timestamp(timestamp):
     If the timestamp cannot be converted, returns None instead.
     """
     try:
-        return datetime.utcfromtimestamp(int(timestamp)).replace(tzinfo=UTC)
+        return datetime.utcfromtimestamp(int(timestamp)).replace(tzinfo=utc)
```

Comment:
Correct.  `django.utils.timezone.UTC` is a minimal version that gets used if pytz isn't importable.  `django.utils.timezone.utc` points to either `pytz.utc` or `django.utils.timezone.UTC`, and is recommended for general use.  pytz has a `utc` and a `UTC`, which are the same object, but pytz.utc is included in the module's `__all__`, and `UTC` isn't, indicating that `utc` is the version intended for general use.